scripts/setup_database.php

```diff
@@ -56,6 +56,8 @@
         error_log('The following required PostgreSQL extensions are missing: ' . implode(', ', $missing_extensions) . ". Please install them.\n");
         exit(1);
     }
+} elseif ($db_driver == 'sqlite') {
+    error_log("Nothing to be done for db driver: {$db_driver}");
```

Comment:
Hello @indridieinarsson 

Thank you for this. But I think instead of writing to error log while there is no error we can just can the else to else if and test if $db_driver != 'sqlite'.

Thank you!